examples/open_pbr_default.mtlx

```diff
@@ -35,8 +35,8 @@
     <input name="coat_roughness_anisotropy" type="float" value="0.0" />
     <input name="coat_ior" type="float" value="1.6" />
     <input name="coat_darkening" type="float" value="1.0" />
-    <input name="thin_film_weight" type="float" value="0.0" />
-    <input name="thin_film_thickness" type="float" value="0" />
+    <input name="thin_film_weight" type="float" value="0" />
```

Comment:
Not that it matters, but it looks like the example is inconsistent when it comes to including or not the `.0`.
In the specification and in the parameter list in particular, I was careful to remain concise for readability, but examples are a different context.